spec/unit/installer/user_project_integrator/target_integrator_spec.rb

```diff
@@ -49,16 +49,6 @@ module Pod
           @target_integrator.integrate!
         end
 
-        it 'fixes the copy resource scripts of legacy installations' do
-          @target_integrator.integrate!
-          target = @target_integrator.send(:native_targets).first
-          phase_name = @copy_pods_resources_phase_name
-          phase = target.shell_script_build_phases.find { |bp| bp.name == phase_name }
-          phase.shell_script = %("${PODS_ROOT}/Pods-resources.sh"\n)
-          @target_integrator.integrate!
-          phase.shell_script.strip.should == '"${PODS_ROOT}/Target Support Files/Pods/Pods-resources.sh"'
-        end
```

Comment:
The "legacy installations" here refers to pre-0.34, I think we can drop any special handling for that by now